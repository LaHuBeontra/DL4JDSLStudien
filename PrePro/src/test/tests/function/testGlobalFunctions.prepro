function main() {
	import vec3 p1, vec3 p2;

	vec3 result1 = testGlobalFunctionReturnFirstArgument(p1);
	vec3 result2 = testGlobalFunctionReturnSecondArgument(p1,p2);
	const amountTimeElements = amountTimeElements(p1);

	export p1, p2, result1, result2, amountTimeElements;
}
