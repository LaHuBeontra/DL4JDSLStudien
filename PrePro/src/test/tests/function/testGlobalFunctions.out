DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='p1', variableShape=[4, 3], startColumn=0, length=3}, DataSetEntry{variableName='p2', variableShape=[4, 3], startColumn=3, length=3}, DataSetEntry{variableName='result1', variableShape=[4, 3], startColumn=6, length=3}, DataSetEntry{variableName='result2', variableShape=[4, 3], startColumn=9, length=3}],
dataSet=
[[42.00,  0.00,  0.00,  43.00,  0.00,  0.00,  42.00,  0.00,  0.00,  43.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00,  44,662.00,  0.00,  0.00,  1.00,  0.00,  0.00,  44,662.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00,  6.00,  0.00,  0.00,  5.00,  0.00,  0.00,  6.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00,  7.00,  0.00,  0.00,  6.00,  0.00,  0.00,  7.00,  0.00,  0.00]]

constants={amountTimeElements=4.00}

calculated_variables=
p1=[[42.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00]]
p2=[[43.00,  0.00,  0.00],  
 [44,662.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00],  
 [7.00,  0.00,  0.00]]
amountTimeElements=4.00
result2=[[43.00,  0.00,  0.00],  
 [44,662.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00],  
 [7.00,  0.00,  0.00]]
result1=[[42.00,  0.00,  0.00],  
 [1.00,  0.00,  0.00],  
 [5.00,  0.00,  0.00],  
 [6.00,  0.00,  0.00]]
}